.travis.yml

```diff
@@ -28,3 +39,9 @@ script:
 
 after_success:
   - coveralls
+  - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
it's just a login procedure, we need to call it before pushing images. Anyway I'll move it because we'll need in future.